app/assets/javascripts/megatron/utils/text-helpers.js

```diff
@@ -50,7 +50,7 @@ var TextHelpers = {
     }
     Array.prototype.forEach.call(document.querySelectorAll('textarea:not(.no-auto-size)'), autoHeight)
 
-    bean.on(document.querySelector('body'), 'keyup', 'textarea', function(event){
+    bean.on(document.querySelector('body'), 'keyup, toggler:show', 'textarea', function(event){
```

Comment:
cool, let's fix that then and also version bump the gem like here: https://github.com/compose-ui/megatron.rb/commit/639dd8295cfec26d63448936687ba921c278e251